### PR TITLE
Implement function to get pull mirror details from a project

### PR DESCRIPTION
**Pull request by timofurrer**

### CODE REVIEW

File: projects.go [hunk 1]
```diff
@@ -1955,6 +1955,43 @@ func (s *ProjectsService) ChangeAllowedApprovers(pid interface{}, opt *ChangeAll
 	return pa, resp, err
 }
 
+// ProjectPullMirrorDetails represent the details of the configuration pull mirror and its update status
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/projects.html#get-a-projects-pull-mirror-details
```
Comment (svanharmelen) [inline projects.go:1960]:
```suggestion
// ProjectPullMirrorDetails represent the details of the configuration pull
// mirror and its update status.
//
// GitLab API docs:
// https://docs.gitlab.com/ee/api/projects.html#get-a-projects-pull-mirror-details
```

File: projects.go [hunk 2]
```diff
@@ -1955,6 +1955,43 @@ func (s *ProjectsService) ChangeAllowedApprovers(pid interface{}, opt *ChangeAll
...
+	LastSuccessfulUpdateAt *time.Time `json:"last_successful_update_at"`
+	LastUpdateAt           *time.Time `json:"last_update_at"`
+	LastUpdateStartedAt    *time.Time `json:"last_update_started_at"`
+	UpdateStatus           string     `json:"update_status"`
+	URL                    string     `json:"url"`
+}
+
+// GetProjectPullMirrorDetails returns the pull mirror details
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/projects.html#get-a-projects-pull-mirror-details
```
Comment (svanharmelen) [inline projects.go:1973]:
```suggestion
// GetProjectPullMirrorDetails returns the pull mirror details.
//
// GitLab API docs:
// https://docs.gitlab.com/ee/api/projects.html#get-a-projects-pull-mirror-details
```

File: projects.go [hunk 3]
```diff
@@ -1955,6 +1955,43 @@ func (s *ProjectsService) ChangeAllowedApprovers(pid interface{}, opt *ChangeAll
...
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("projects/%s/mirror/pull", PathEscape(project))
+
+	req, err := s.client.NewRequest(http.MethodGet, u, nil, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	model := new(ProjectPullMirrorDetails)
```
Comment (svanharmelen) [inline projects.go:1986]: `model` is kind of a weird name (IMHO)... Maybe just `pmd` to keep it more consistent with other namings?

**Review by svanharmelen — CHANGES_REQUESTED** [review]

File: projects.go (hunk 3 above)
Comment (timofurrer) [reply projects.go]: done

**Review by svanharmelen — APPROVED** [review]

**Pull request merged**